drivers/test.go

```diff
@@ -550,31 +557,36 @@ func (r *RewardSummary) For(a address.Address) abi_spec.TokenAmount {
 }
 
 func (td *TestDriver) GetRewardSummary() *RewardSummary {
-	var rst reward_spec.State
-	td.GetActorState(builtin_spec.RewardActorAddr, &rst)
-	rewards := make(map[address.Address]abi_spec.TokenAmount)
-	// Traverse map keyed by miner address.
-	var r cbg.CborCid
-	err := adt_spec.AsMap(AsStore(td.State()), rst.RewardMap).ForEach(&r, func(key string) error {
-		keyAddr, err := address.NewFromBytes([]byte(key))
-		require.NoError(td.T, err)
-
-		// Traverse array of reward entries.
-		sum := big_spec.Zero()
-		var rw reward_spec.Reward
-		err = adt_spec.AsArray(AsStore(td.State()), cid.Cid(r)).ForEach(&rw, func(i int64) error {
-			sum = big_spec.Sub(big_spec.Add(sum, rw.Value), rw.AmountWithdrawn)
+	// FIXME the reward actor's responsibilities were stripped down, this method needs to be adjusted to reflect that
```

Comment:
need to fix before merging

Reply:
The rewards earned by a miner have moved to the miner actor as `LockedFunds`. I think inspecting that was the primary use of this structure.

We might still want some summary of the reward actor's new state, but we can start without it.